app/assets/stylesheets/components/lesson/lesson_content.scss

```diff
@@ -68,11 +68,6 @@
     }
   }
 
-  ol {
-    list-style: decimal;
```

Comment:
I think we still need to define this. The Tailwind base directive we enabled last week is basically a reset and it strips out _all_ list styling unfortunately 😫

Reply:
aight cando